app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :posts
+  belongs_to :users
```

Comment:
- [ ] Same as here, use the singular form. 

Optional: You can also look at how to use `class_name`